Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -76,9 +115,7 @@ Manual Verification of Setup
     * Validate that a local node in the source cluster can reach the IP
       specified in the ``hostAliases`` section. The ``clustermesh-secrets``
       secret contains a configuration file for each remote cluster, it will
-      point to a logical name representing the remote cluster:
-
-      .. code-block:: yaml
+      point to a logical name representing the remote cluster::
```

Comment:
Is the snippet here YAML? If yes, then we don't need to change it. If no, this is fine :+1: